js/components/charts/widget.vue

```diff
@@ -119,91 +124,80 @@ export default {
         }
     },
     computed: {
-        series: function() {
-            let series = this.y.map((item) => {
-                return item.id;
-            });
-            let raw = this.metrics.timeserie(series);
-            let data = {
-                labels: raw.map((item) => {
-                    return moment(item.date).format('L');
-                }),
-
+        series() {
+            const series = this.y.map(item => item.id);
+            const raw = this.metrics.timeserie(series);
+            return {
+                labels: raw.map(item => moment(item.date).format('L')),
                 datasets: this.y.map((serie, idx) => {
-                    let dataset = {label: serie.label};
-                    let color = serie.color || COLORS[idx];
+                    const dataset = {label: serie.label};
+                    const color = serie.color || COLORS[idx];
                     dataset.fillColor = this.toRGBA(color, .5);
                     dataset.strokeColor = color;
                     dataset.pointColor = color;
                     // datasetpointStrokeColor: "#c1c7d1",
                     dataset.pointHighlightFill = '#fff';
                     dataset.pointHighlightStroke = color;
-                    dataset.data = raw.map((item) => {
-                        return item[serie.id];
-                    });
+                    dataset.data = raw.map(item => item[serie.id]);
                     return dataset;
                 })
             };
-
-            return data;
         }
     },
-    components: {
-        box: require('components/containers/box.vue')
-    },
-    ready: function() {
+    components: {Box},
```

Comment:
Vue performs Camel/Kebab cas to dash transform as HTML is case insensitive but allows dashes and JS doesn't allows dashes and is case sensitive:
- `Box` -> `box
- `TwoParts` -> `two-parts`